javascript/packages/orchestrator/src/chain-spec.ts

```diff
@@ -374,7 +376,21 @@ export async function changeGenesisConfig(specPath: string, updates: any) {
 }
 
 export async function addBootNodes(specPath: string, addresses: string[]) {
-  const chainSpec = readAndParseChainSpec(specPath);
+  let chainSpec;
+  try {
+    chainSpec = readAndParseChainSpec(specPath);
+  } catch (e: any) {
+    if (e.code !== "ERR_FS_FILE_TOO_LARGE") throw e;
+
+    // can't customize bootnodes
+    console.log(
```

Comment:
Since this is an error, it should be red instead of yellow, and maybe we should also print it

Reply:
Is not an error, the file is too big so we can't customize but we can deploy the namespace.